Sources/KituraMarkdown/KituraMarkdown.swift

```diff
@@ -101,21 +107,26 @@ public class KituraMarkdown: TemplateEngine {
 
     /// Generate HTML from a String containing text marked up in Markdown.
     ///
+    /// - Parameter from: The String containing markdown text in UTF-8
     /// - Returns: A String containing an HTML representation of the text marked up
     ///            using Markdown.
     public static func render(from: String) -> String {
-        let md = from.data(using: .utf8)
-        return  md != nil ? KituraMarkdown.render(from: md!) : ""
+        guard let md = from.data(using: .utf8) else {
+            return ""
+        }
+
+        return KituraMarkdown.render(from: md)
     }
 
     /// Generate HTML from a String containing text marked up in Markdown.
     ///
+    /// - Parameter from: The String containing markdown in UTF-8
+    /// - Parameter pageTemplate: The HTML page template used
     /// - Returns: A String containing an HTML representation of the text marked up
     ///            using Markdown.
     public static func render(from: String, pageTemplate: String) -> String {
```

Comment:
Yes but it's also a breaking change because the external parameter name is now different